Doc/library/sqlite3.rst

```diff
@@ -1054,9 +1054,10 @@ Blob Objects
 
 .. class:: Blob
 
-   A :class:`Blob` instance is a :term:`file-like object` that can read and write
-   data in an SQLite :abbr:`BLOB (Binary Large OBject)`.  Call ``len(blob)`` to
-   get the size (number of bytes) of the blob.
+   A :class:`Blob` instance is a :term:`file-like object` with
+   :term:`mapping` support,
```

Comment:
I'm OK with using the mapping slots internally, but I still think it's confusing to document this behavior as "mapping" support. The publicly visible behavior is sequence behavior (e.g., slice assignment), not mapping behavior.

Reply:
How about this:

`The blob data can be accessed and modified directly using indices and slices.`

(Of course with ```:term:`slice` ```...)

Reply:
Sure! Maybe also add some examples to the example file.